html-forms/html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="email" id="email" name="email" />
+      <div>
+        <label for="why-is-coding-cool">Why is coding cool?</label>
+      </div>
+      <select id="why-is-coding-cool">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
```

Comment:
These should all have a `value` attribute with a proper corresponding value (the string assigned to the `value` attribute is what gets sent to the server on a form submission)

ex: `<option value="interesting">Because it&apos;s interesting</option>`